scripts/irods/test/test_ireg.py

```diff
@@ -209,3 +209,11 @@ def test_ireg_double_slashes__issue_3658(self):
         self.admin.assert_icommand('iscan {0}'.format(os.path.abspath(dirname)))
         shutil.rmtree(os.path.abspath(dirname), ignore_errors=True)
 
+    def test_ireg_file_with_unresolvable_owner__issue_4040(self):
+        filename = '../test/unresolvable_uid_testfile__issue_4040'
+        fullpath = os.path.abspath(filename)
+        # if file is there (should be for a package installation)
+        # attempt the ireg, should pass now that 4040 is fixed
+        if os.path.isfile(fullpath):
+            self.admin.assert_icommand('ireg {0} {1}'.format(fullpath, self.admin.session_collection+'/'+os.path.basename(filename)))
```

Comment:
Minor/optional: Could you use a list here to avoid the use of format/{0}{1}? Something like:
```
['ireg', fullpath, self.admin.session_collection+'/'+os.path.basename(filename))]
```

Reply:
sure....  is avoiding format() something to aspire to?